src/main/java/com/vagasti/backend/inscricoes/InscricaoController.java

```diff
@@ -0,0 +1,39 @@
+package com.vagasti.backend.inscricoes;
+
+import com.vagasti.backend.security.CustomUserDetails;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(produces = "application/json")
+public class InscricaoController {
+  private final InscricaoService inscricaoService;
+
+  public InscricaoController(InscricaoService inscricaoService) {
+    this.inscricaoService = inscricaoService;
+  }
+
+  @PostMapping("/public/novaInscricao")
```

Comment:
End point não pode ser público
```suggestion
  @PostMapping("/novaInscricao")
```